daemon/containerd/image_exporter.go

```diff
@@ -39,7 +40,7 @@ func (i *ImageService) PerformWithBaseFS(ctx context.Context, c *container.Conta
 // outStream is the writer which the images are written to.
 //
 // TODO(thaJeztah): produce JSON stream progress response and image events; see https://github.com/moby/moby/issues/43910
-func (i *ImageService) ExportImage(ctx context.Context, names []string, outStream io.Writer) error {
+func (i *ImageService) ExportImage(ctx context.Context, names []string, outStream io.Writer) (err error) {
```

Comment:
[Why are you naming the result parameter?](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters) I don't see any new deferred closures which reference it.

Reply:
I used it at one point, then reverted that particular change and forgot about this one thx